docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,63 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Create TraceQL queries in Grafana
 
-You can use the TraceQL viewer and query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+You can use compose TraceQL queries in Grafana and Grafana Cloud using the Search in Explore and the TraceQL viewer and query editor in the Tempo data source.
+Both of these methods let you build queries and drill-down into result sets.
+
+To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
 
 {{% admonition type="note" %}}
-To use the TraceQL query editor in Grafana, you need to enable the `traceqlEditor` feature flag. This feature is available starting in Grafana 9.3.2. The query editory is available automatically in Grafana Cloud.
+To use the TraceQL query editor in Grafana 9.3.2 and newer, you need to enable the `traceqlEditor` feature flag.
+
+To enable the Trace Search in Grafana 10 and newer, you need to enable the `traceqlSearch` feature flag.
+
+The query editor and search are available in Grafana Cloud without enabling a feature flag.
 {{% /admonition %}}
 
+## Create TraceQL queries using the query editor
+
+The Tempo data source’s query editor helps you query and display traces from Tempo in **Explore**.
+
+To access the query editor, follow these steps:
+
+1. Sign into Grafana or Grafana Cloud.
+1. Select your Tempo data source.
+1. From the menu, choose **Explore** and select the **TraceQL** tab.
+1. Start your query on the text line by entering `{{`. For help with TraceQL syntax, refer to the [Construct a TraceQL query documentation]({{< relref "./_index.md" >}}).
+1. Once you have finished your query, select **Run query**.
+
 ![Query editor showing request for http.method](/static/img/docs/tempo/query-editor-http-method.png)
 
-Using the query editor, you can use the editor’s autocomplete suggestions to write queries. The editor detects span sets to provide relevant autocomplete options. It uses regular expressions (regex) to detect where it is inside a spanset and provide attribute names, scopes, intrinsic names, logic operators, or attribute values from Tempo's API, depending on what is expected for the current situation.
+### Query by TraceID
+
+To query a particular trace:
+
+1. From the menu, choose **Explore** and select the **TraceQL** tab.
+1. Enter the trace’s ID into the query field.
```

Comment:
Can we provide an example, is it between `{...}` brackets or without?

Reply:
I confirmed that adding a trace ID directly on the search field on the TraceQL tab and pressing Run query returns data for the trace ID. I have a screenshot I can add to this section, if you think that would be helpful.